pkg/doc/jwt/claims.go

```diff
@@ -0,0 +1,123 @@
+/*
+ *
+ *  * Copyright SecureKey Technologies Inc. All Rights Reserved.
```

Comment:
the license header doesn't look normal.

Reply:
Ah. Maybe I placed a wrong template for the license setting in my IDE. Thanks for the catch!